sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -4055,19 +4055,28 @@ class AstBuilder extends DataTypeAstBuilder with SQLConfHelper with Logging {
   }
 
   /**
-   * Create a [[ShowTableExtended]] command.
+   * Create a [[ShowTablesExtended]] or [[ShowTablePartition]] command.
    */
   override def visitShowTableExtended(
```

Comment:
Because we change `ShowTableExtended` to `ShowTablesExtended`, shall we update the name in g4 ?

Reply:
@cloud-fan 
Do we need this related modification?
Because renaming the name to `ShowTablesExtended` or `ShowTablePartition` does not seem to convey the meaning of the following two commands at the same time.
https://github.com/apache/spark/blob/57e73dafeb9ffd288e0ed4ad76b9549e7da33ad7/sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4#L176-L177

Reply:
Let's not rename the parser rule now. It matches both commands.